platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitService.java

```diff
@@ -252,6 +254,19 @@ public MergeResult merge(MergeRequest request) throws ApiException {
         }
     }
 
+    @Path("rebase")
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public RebaseResponse rebase(RebaseRequest request) throws ApiException {
+        GitConnection gitConnection = getGitConnection();
```

Comment:
You can write
try (GitConnection gitConnection = getGitConnection()) {
    gitConnection.rebase(request);
}